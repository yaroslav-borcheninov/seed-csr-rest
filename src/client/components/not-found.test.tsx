import React from "react"
import { render } from "react-testing-library"

import NotFound from "./not-found"

describe("NotFound", () => {
  it("should render", () => {
    expect(render(<NotFound />).container).toMatchSnapshot()
  })
})
