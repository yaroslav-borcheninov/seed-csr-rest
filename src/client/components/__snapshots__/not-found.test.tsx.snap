// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render 1`] = `
<div>
  <div
    class="sc-bwzfXH eQJZQL"
  >
    Not found...
  </div>
</div>
`;
